packages/frontend/src/components/SpacePanel/index.tsx

```diff
@@ -298,7 +201,7 @@ export const SpacePanel: React.FC<Props> = ({ space }) => {
                     setIsCreateDashboardOpen(false);
                 }}
             />
-        </Stack>
+        </>
```

Comment:
extra fragment wrapper

Reply:
im happy to get rid of this file if space panel is not referenced more than once

Reply:
sounds good! its only refrenced here but i was afraid itd make the space page too big 
if its ok with you though ill scrap it

Reply:
ok i'll scrap it in 2nd PR cause I'm making changes to the ResourceView component there